src/m365/planner/PlanDetails.ts

```diff
@@ -0,0 +1,14 @@
+export interface PlanDetails {
```

Comment:
Instead of introducing our own type, let's use the typings from the @microsoft/microsoft-graph-types package.

Reply:
done, thanks for review, helpful for improving and learning.